Day1/data.csv

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
@anantmishra-successive This is o/p file. this is not part of the solution, so it can be ignored. This can be generated by running the code.

Reply:
![ignore](https://github.com/anantmishra-successive/Backend_Training/assets/162098832/d859672c-20a3-43b3-b69a-522a5890890f)

Reply:
Ok sir I added this file in .gitignore